types/snowflake-sdk/index.d.ts

```diff
@@ -163,6 +163,21 @@ export interface SnowflakeError extends SnowflakeErrorExternal {
     externalize?: () => SnowflakeErrorExternal;
 }
 
+export enum validFetchAsStringType {
+    STRING = 'STRING',
+    BOOLEAN = 'BOOLEAN',
+    NUMBER = 'NUMBER',
+    DATE = 'DATE',
+    JSON = 'JSON',
+}
```

Comment:
Why do we need this type?
```suggestion
```

Reply:
because the validator require me to convert Array<Type> -> Type[].
So i tried to create a specific allowed type for this array.